GVFS/GVFS/CommandLine/CloneVerb.cs

```diff
@@ -662,6 +662,7 @@ private void CheckNotInsideExistingRepo(string normalizedEnlistmentRootPath)
             return new Result(true);
         }
 
+        // TODO(Linux), TODO(Mac): either adjust to "git" or remove entirely

```

Comment:
Good call! An equivalent shell script could be:

```
#!/bin/sh

echo This repo was cloned using VFS for Git, and the git repo is in the 'src' directory
echo Switching you to the 'src' directory and rerunning your git command

cd src
git "$@"
```

Reply:
Of course, there are some special characters in the `git.cmd` that probably do color stuff, but I'm too lazy to figure out how to do that in POSIX.

Reply:
Well, and the other consideration is that where Windows will (I believe; I'm no expert) search the [current directory first](https://docs.microsoft.com/en-us/previous-versions//cc723564(v=technet.10)#command-search-sequence) for an executable, on Unix systems, unless someone happens to have put `.` in their `PATH` (which is generally frowned upon), that's unlikely.

So I'm not sure if there's really any value to this kind of "redirector" script on Mac or Linux, even if we named it `git`.  Plus I feel like it would be confusing for folks who know there needs to be a custom `git` binary to work with VFSForGit if we *also* have a script named the same thing, but which isn't actually that binary.

Anyway, I just wanted to flag it because it was something I bumped into along the road to `GVFS.Platform.Linux`.  :-)

Reply:
Good point! Then, it's worth leaving as a TODO until we have a concrete decision about what to do with this step. Apparently, it was an important feature for users confused about where to go after a `gvfs clone` step.

Reply:
I could also open an issue on this to track it for future work, if that's useful.

Reply:
>I could also open an issue on this to track it for future work, if that's useful.

I think it's worthwhile to have an issue tracking this, could you file one?

Reply:
Done as #1138 -- thanks!